receiver/gitproviderreceiver/metadata.yaml

```diff
@@ -13,40 +13,85 @@ status:
 # this might need to be unique per sub receiver implementation
 resource_attributes:
   organization.name:
-      enabled: true
-      description: Git Organization or Project Name
-      type: string
+    enabled: true
+    description: Git Organization or Project Name
+    type: string
   git.vendor.name:
-      enabled: true
-      # github, gitlab, bitbucket, gittea
-      description: The name of the Git vendor/provider (ie. GitHub / GitLab)
-      type: string
+    enabled: true
+    # github, gitlab, bitbucket, gittea
+    description: The name of the Git vendor/provider (ie. GitHub / GitLab)
+    type: string
 
-## Attritbutes that will be uncommented when the rest of the metrics are added
+## Attributes that will be uncommented when the rest of the metrics are added
 attributes:
   repository.name:
-      description: The full name of the Git repository
-      type: string
+    description: The full name of the Git repository
```

Comment:
```suggestion
    description: The name of the Git repository
```

Reply:
I believe it's just a matter of whether or not the owner of the repo is included. For example, in the case of the current repo we're in, the name would be `opentelemetry-collector-contrib` whereas the full name is `open-telemetry/opentelemetry-collector-contrib`.